mkt/extensions/tests/test_views.py

```diff
@@ -1285,7 +1285,7 @@ def test_post_logged_in_with_rights(self):
         }))
         eq_(response.status_code, 201)
         data = response.json
-        eq_(data['size'], 268)  # extension.zip size in bytes.
+        eq_(data['size'], 526)  # extension.zip size in bytes.
```

Comment:
Yeah, I should have done that the last couple times I modified it, I'll do it now. Same for the hash.